Sming/Arch/Esp32/Components/esp32/component.mk

```diff
@@ -298,13 +311,9 @@ FLASH_BOOT_CHUNKS		:= 0x1000=$(FLASH_BOOT_LOADER)
 
 SDK_DEFAULT_PATH := $(COMPONENT_PATH)/sdk
 
-##@Partitions
-
-SDK_PARTITION_PATH := $(SDK_DEFAULT_PATH)/partitions
-
 ##@SDK
 
-SDK_PROJECT_PATH := $(COMPONENT_PATH)/project.$(ESP_VARIANT)
+SDK_PROJECT_PATH := $(COMPONENT_PATH)/project/$(ESP_VARIANT)
```

Comment:
Adding `/$(BUILD_TYPE)` to the end of this path fixes the issue switching between debug/release builds. We get a separate project instance for each.